### ISSUE TITLE
Get someone to be the treasurer

### ISSUE THREAD

**Issue by hellozee** [ILUGD/volunteers-tasks#21]
Since we would be taking in the domain, we would need someone to collect money for the renewal. And of course in future we plan to host other things too on the domain, so for that we need to maintain our expenses, correctly.

**Comment by insaaniManav** [comment]
@miphilomath can you please take a look at it

**Comment by xeon-zolt** [comment]
@rajudev  @miphilomath will you be available for a quick discussion on this on volunteer call

**Comment by miphilomath** [comment]
> Since we would be taking in the domain, we would need someone to collect money for the renewal. And of course in future we plan to host other things too on the domain, so for that we need to maintain our expenses, correctly.

Agreed. I suggest we setup a open collective account in the long run, to keep track of the donations/support and the expense.
